lib/simulator-xcode-6.js

```diff
@@ -290,6 +291,10 @@ class SimulatorXcode6 extends EventEmitter {
     if (!this.connectHardwareKeyboard) {
       args.push('-ConnectHardwareKeyboard', '0');
     }
+
+    // Do the 'Touch ID Enroll' key bindings before the Simulator starts
+    await setTouchEnrollKey();
```

Comment:
also remember that this whole behavior will be behind a desired cap, so we shouldn't do it every time. we should only do it if a certain param is passed in to the ios sim constructor.